src/vma/dev/net_device_table_mgr.cpp

```diff
@@ -298,7 +307,7 @@ int net_device_table_mgr::map_net_devices()
 
 // Verify IPoIB is in 'datagram mode' for proper VMA with flow steering operation
 // Also verify umcast is disabled for IB flow
-void net_device_table_mgr::verify_ipoib_mode(struct ifaddrs* ifa)
+bool net_device_table_mgr::verify_ipoib_mode(struct ifaddrs* ifa)
```

Comment:
As i see it, there's no point to continute and create a device if we do not support the chosen ipoib mode. it can either fail elsewhere in an unpredicted way or be offloaded in a later time (which will cause the same behavior as in this PR but harm performance)